### PR TITLE
Add the prpqr-aggregator ci-operator step in registry

### PR DESCRIPTION
**Pull request by droslean**
/cc @bradmwilliams @deads2k 
/cc @openshift/test-platform  

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    mount_path: /var/run/secrets
+  env:
+  - name: GOOGLE_SA_CREDENTIAL_FILE
+  - name: VERIFICATION_JOB_NAME
+  - name: AGGREGATION_ID
+  - name: EXPLICIT_GCS_PREFIX
+  - name: JOB_START_TIME
+  - name: WORKING_DIR
+  documentation: |-
+    Launches the job-run-aggregator to analyze the release analysis jobs launched with a specific aggregation id.
```
Comment (hongkailiu) [inline ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-ref.yaml:26]: Does the document support links? 
NIT: add link to https://github.com/openshift/ci-tools/tree/master/cmd/job-run-aggregator

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/OWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+../../OWNERS
```
Comment (petr-muller) [inline ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/OWNERS:1]: If this is something that our tooling will trigger, then we should own it, not release-controller owners.

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+job-run-aggregator analyze-job-runs \
+  --google-service-account-credential-file ${GOOGLE_SA_CREDENTIAL_FILE} \
```
Comment (petr-muller) [inline ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh:8]:
```suggestion
  --google-service-account-credential-file=${GOOGLE_SA_CREDENTIAL_FILE} \
```

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/OWNERS (hunk 2 above)
Comment (droslean) [reply ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/OWNERS:1]: I'll open a follow-up

**Pull request merged**